src/resdk/tables/variant.py

```diff
@@ -205,71 +256,21 @@ def _data(self) -> List[Data]:
 
         return data
 
-    @property
-    @lru_cache()
-    def variants(self) -> pd.DataFrame:
-        """Return variants table as a pandas DataFrame object."""
-        return self._load_fetch(self.VARIANTS)
-
-    @property
-    @lru_cache()
-    def depth(self) -> pd.DataFrame:
-        """Return depth table as a pandas DataFrame object."""
-        return self._load_fetch(self.DEPTH)
-
-    @property
-    @lru_cache()
-    def depth_a(self) -> pd.DataFrame:
-        """Return depth table for adenine as a pandas DataFrame object."""
-        return self._load_fetch(self.DEPTH_A)
-
-    @property
-    @lru_cache()
-    def depth_c(self) -> pd.DataFrame:
-        """Return depth table for cytosine as a pandas DataFrame object."""
-        return self._load_fetch(self.DEPTH_C)
-
-    @property
-    @lru_cache()
-    def depth_g(self) -> pd.DataFrame:
-        """Return depth table for guanine as a pandas DataFrame object."""
-        return self._load_fetch(self.DEPTH_G)
-
-    @property
-    @lru_cache()
-    def depth_t(self) -> pd.DataFrame:
-        """Return depth table for thymine as a pandas DataFrame object."""
-        return self._load_fetch(self.DEPTH_T)
-
     def _download_qc(self) -> pd.DataFrame:
         """Download sample QC data and transform into table."""
         # No QC is given for variants data - return empty DataFrame
         return pd.DataFrame()
 
-    def strip_amino_acid(self, amino_acid_string, remove_alt=False):
-        """Remove "p." prefix of the amino acid string."""
-        # Example Amino acid change:  "p.Gly12Asp"
-        if pd.isna(amino_acid_string):
-            return ""
-        aa_match = re.match(
-            r"^p?\.?([A-Z][a-z]{2}\d+[A-Z][a-z]{2})$", amino_acid_string
-        )
-        if aa_match:
-            if remove_alt:
-                return aa_match.group(1)[:-3]
-            return aa_match.group(1)
-        return ""
-
-    def construct_index(self, row) -> str:
+    def _construct_index(self, row) -> str:
         """
         Construct index of the variants table.
 
         Index should have the form:
-        <chr>_<position>_<snp_change>_<aminoacid_pos_change>
-        E.g. chr2_1234567_C>T_Gly12Asp
+        <chr>_<position>_<snp_change>
+        E.g. chr2_1234567_C>T
 
-        In cases where nucleotide change or amino-acid change is
-        missing, this info is left out from the index.
+        In cases where nucleotide change is missing, this info is left out from
```

Comment:
This can never happen anymore, since the variant is only reported if it was detected.

Reply:
Cool!